### PR TITLE
Add node-semver

### PR DESCRIPTION
**Pull request by windelbouwmanbosch**

### CODE REVIEW

**Comment by windelbouwmanbosch** [comment]
Okay, so I suggest to name is python-node-semver? Since there also exists a package semver for python.

**Comment by dougalsutherland** [comment]
`python-node-semver` is fine with me, and probably better than `python-semver` given the python `semver` package. Note though that `node-semver`'s package you import is also called `semver`, so the two packages conflict (not that conda can note that). :/

File: recipes/node-semver/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+about:
+  home: https://github.com/podhmo/python-semver
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: port of node-semver
+  dev_url: https://github.com/podhmo/python-semver
+
+extra:
+  recipe-maintainers:
+    - windelbouwman
```
Comment (dougalsutherland) [inline recipes/node-semver/meta.yaml]: Is this also you? Why the two IDs?

Comment (windelbouwmanbosch) [reply recipes/node-semver/meta.yaml]: Yes that is also me. The two accounts are redundant.

File: recipes/python-node-semver/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```
Comment (dougalsutherland) [inline recipes/python-node-semver/meta.yaml]: `setuptools` isn't actually needed at runtime, from what I can tell. I'll drop it for you and merge.

**Comment by dougalsutherland** [comment]
Merged! Thanks @windelbouwmanbosch.

**Pull request merged**